### PR TITLE
[12.0] [FIX] [hr_employee_service_contract]

### PR DESCRIPTION
**Pull request by YuriyGural**
Сorrect filling of the fields "service_start_date", "service_termination_date".
It depends on the status change from "New" to "Running", and also if the "End date" field in one of the contracts is empty.

### CODE REVIEW

**Comment by etobella** [comment]
Tests are failing due to your change, also, why did you change the function?

File: hr_employee_service_contract/models/hr_employee.py [hunk 1]
```diff
@@ -42,22 +42,31 @@ def _compute_first_contract_id(self):
...
+            employee.last_contract_id = self._get_last_contract_id(employee.id)
+    
     @api.multi
     @api.onchange('service_hire_date')
     def _onchange_service_hire_date(self):  # pragma: no cover
         # Do nothing
         pass
 
+    @api.multi
+    def _get_last_contract_id(self, employee_id):
```
Comment (alexey-pelykh) [inline hr_employee_service_contract/models/hr_employee.py:58]: @YuriyGural if you're changing the behaviour, please describe the change as well as provide tests for it

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

File: hr_employee_service_contract/models/hr_employee.py (hunk 1 above)
Comment (YuriyGural) [reply hr_employee_service_contract/models/hr_employee.py:58]:
```
    def test_6(self):
        employee = self.SudoEmployee.create({
            'name': 'Employee #6',
        })
        contract_1 = self.Contract.create({
            'name': 'Employee #6 Contract #1',
            'employee_id': employee.id,
            'wage': 5000.0,
            'state': 'open',
            'date_start': self.today - relativedelta(years=5),
            'date_end': self.today - relativedelta(years=1),
        })
        contract_2 = self.Contract.create({
            'name': 'Employee #6 Contract #2',
            'employee_id': employee.id,
            'wage': 6000.0,
            'state': 'draft',
            'date_start': self.today - relativedelta(years=2),
            'date_end': False,
        })

        self.assertEqual(
            employee.service_termination_date,
            self.today - relativedelta(years=1)
        )

        contract_2.write({'state': 'open'})
        self.assertEqual(
            employee.service_termination_date,
            False
        )
```

Comment (alexey-pelykh) [reply hr_employee_service_contract/models/hr_employee.py:58]: https://github.com/OCA/hr/blob/12.0/hr_employee_service/tests/test_hr_employee_service.py

**Pull request closed without merging**